### PR TITLE
allow specifying default encoding for publisher;

### PR DESCRIPTION
**Pull request by DenisBiryukov91**
allow specifying default encoding for publisher;
put clean up;

### CODE REVIEW

File: zenohpico.pc [hunk 1]
```diff
@@ -1,8 +1,8 @@
-prefix=/usr/local
+prefix=/Users/denisbiryukov/repositories/work/local
```
Comment (sashacmc) [inline zenohpico.pc:1]: Please revert this change

**Review by sashacmc — APPROVED** [review]
Small note, others LGTM

File: zenohpico.pc (hunk 1 above)
Comment (DenisBiryukov91) [reply zenohpico.pc]: fixed

**Review by milyin — APPROVED** [review]
